docs/platform/one-click/deploy-postresql-with-one-click-apps/index.md

```diff
@@ -0,0 +1,155 @@
+
+
+## Getting Started after Deployment
+
+### Access PostgreSQL
+
+After PostgreSQL has finished installing, you will be able to access PostgreSQL from the console via ssh with your Linode's IPv4 address:
+
+1.  [SSH into your Linode](/docs/getting-started/#connect-to-your-linode-via-ssh) and [create a limited user account](/docs/security/securing-your-server/#add-a-limited-user-account).
+
+1.  Log out and log back in as your limited user account.
+
+1.  Update your server:
+
+        sudo apt-get update && apt-get upgrade
+
+## Using PostgreSQL
+
+### Modify the Postgres Users
+
+By default, PostgreSQL will create a Linux user named `postgres` to access the database software.
+
+{{< caution >}}
+The `postgres` user should not be used for other purposes (e.g. connecting to other networks). Doing so presents a serious risk to the security of your databases.
+{{< /caution >}}
+
+1.  Change the `postgres` user's Linux password:
+
+        sudo passwd postgres
+
+2.  Issue the following commands to set a password for the `postgres` database user. Be sure to replace `newpassword` with a strong password and keep it in a secure place.
+
+        su - postgres
+        psql -d template1 -c "ALTER USER postgres WITH PASSWORD 'newpassword';"
+
+    This user is distinct from the `postgres` Linux user. The Linux user is used to access the database, and the PostgreSQL user is used to perform administrative tasks on the databases.
+
+    The password set in this step will be used to connect to the database via the network. Peer authentication will be used by default for local connections. See the [Secure Local PostgreSQL Access section](#secure-local-postgresql-access) for information about changing this setting.
+
+### Create a Database
+
+Run the commands in this section as the `postgres` Linux user.
+
+1.  Create a sample database called `mytestdb`:
+
+        createdb mytestdb
+
+2.  Connect to the test database:
+
+        psql mytestdb
+
+3.  You will see the following output:
+
+        psql (12.2 (Debian 12.2-2.pgdg90+1))
+        Type "help" for help.
+
+        mytestdb=#
+
+    This is the PostgreSQL client shell, in which you can issue SQL commands. To see a list of available commands, use the `\h` command. You may find more information on a specific command by adding it after `\h`.
+
+### Create Tables
+
+This section contains examples which create a test database with an employee's first and last name, assigning each a unique key. When creating your own tables, you may specify as many parameters (columns) as you need and name them appropriately. Run the commands in this section from the PostgreSQL client shell, that you opened to create `mytestdb` database.
+
+1.  Create a table called "employees" in your test database:
+
+        CREATE TABLE employees (employee_id int, first_name varchar, last_name varchar);
```

Comment:
Should employee_id be SERIAL PRIMARY KEY? Just to show how to auto increment? Up to you to decide if this is necessary.

Reply:
fixed